src/Domain.jl

```diff
@@ -1585,6 +1585,16 @@ end
 end
 export calcdomainderivatives!
 
+@inline function jacobianyefficiencyderiv!(jac::S,domain::Union{ConstantTPDomain,ConstantPDomain,ParametrizedTPDomain,ParametrizedPDomain},kinetics::AbstractFalloffRate,efficiencies::Dict{Int64,Float64},rxnarray::Array{Int64,2},rxnind::Int64,Vind::Int64,cs::Array{Float64,1},T::Float64,V::Float64,C::Float64,Ceff::Float64,Kc::Float64) where {S<:AbstractArray}
```

Comment:
Can you squash the two non-kinetics object specific jacobian efficiencies commits here and then put the three related to the kinetics objects together in a second commit?